helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/CodeTransformations.java

```diff
@@ -74,6 +74,11 @@
   private static final Pattern IMPORT_PATTERN = Pattern.compile("import\\s+(.*);");
   private static final Pattern ENUM_PATTERN = Pattern.compile("public enum (\\w+)");
   private static final Pattern ANNOTATION_PATTERN = Pattern.compile("\\s*@.*");
+  private static final Pattern CATCH_FIELD_EXCEPTION_START_PATTERN = Pattern.compile("catch \\(org.apache.avro.AvroMissingFieldException e\\) \\{");
+  private static final Pattern CATCH_FIELD_EXCEPTION_END_PATTERN = Pattern.compile("throw new org.apache.avro.AvroRuntimeException\\(e\\);\\s*}");
```

Comment:
How do we know that there will always be a `throw new AvroRuntimeException`  at the end of the clause list? 
Maybe just remove the catch clause of the `AvroMissingFieldException` ?

Reply:
just removing the top catch clause would mean that AvroMissingFieldExceptions would always be double-wrapped in an AvroRuntimeException, which is a change of behaviour under 1.9/10. 

we know this pattern holds for 1.9 and 1.10. if something new happens in 1.11+ we'll have to adjust. the theoretically correct thing to do is stop mutilating code post-avro and throw avro code0gen out the window entirely and replace it with our own, but thats a longer-term todo